pennylane/init.py

```diff
@@ -493,11 +493,15 @@ def cvqnn_layer_normal(n_wires, mean=0, std=1, mean_active=0, std_active=0.1, se
 
 
 def interferometer_uniform(n_wires, low=0, high=2 * pi, seed=None):
-    r"""Creates a list of three parameter arrays for :func:`~.Interferometer`, drawn from a uniform distribution.
+    r"""Returns a list of three arrays of the form ``[theta, phi, varphi]`` where:
 
-    The shape of the arrays is either ``(n_wires,)`` or ``(n_wires*(n_wires-1)/2,)``.
+    * ``theta`` is the list of beam splitter transmittivity angles, of size ``(n_wires*(n_wires-1)/2, )``
 
-    The parameters are initialized uniformly from the interval ``[low, high]``.
+    * ``phi`` is the list of beam splitter phases, of size ``(n_wires*(n_wires-1)/2, )``
```

Comment:
here it says "list", but below it says "array" (same for other angles). Which is it?

Reply:
The function returns a list of 3 arrays. it should be `array` here. will fix it